### PR TITLE
chore: add notification bot support for vs

### PR DESCRIPTION
**Pull request by xiaolang124**
chore: add notification bot support for vs

### CODE REVIEW

File: templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    # Output: following environment variable will be persisted in current environment's .env file.
+    # TEAMS_APP_ID: the id of Teams app
+
+provision:
+  - uses: botAadApp/create # Creates a new AAD app for bot if BOT_ID environment variable is empty
+    with:
+      name: notification-bot
+    # Output: following environment variable will be persisted in current environment's .env file.
+    # BOT_ID: the AAD app client id created for bot
+    # BOT_PASSWORD: the AAD app client secret created for bot
```
Comment (kuojianlu) [inline templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml:16]:
```suggestion
    # SECRET_BOT_PASSWORD: the AAD app client secret created for bot
```

File: templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        BOT_ID: ${{BOT_ID}}
+        BOT_PASSWORD: ${{SECRET_BOT_PASSWORD}}
+
+  - uses: botFramework/createOrUpdateBot # Create or update the bot registration on dev.botframework.com
+    with:
+      botId: ${{BOT_ID}}
+      name: notification-bot
+      messagingEndpoint: ${{TUNNEL_ENDPOINT}}/api/messages
+      description: ""
+      iconUrl: ""
```
Comment (kuojianlu) [inline templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml:31]: Remove `iconUrl` arg.

File: templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml (hunk 1 above)
Comment (xiaolang124) [reply templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml]: Done.

File: templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml (hunk 2 above)
Comment (xiaolang124) [reply templates/scenarios/csharp/notification-http-timer-trigger/teamsfx/app.local.yml]: Done.

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**